.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb/base"
```

Comment:
In order to use _eslint-config-airbnb-base_ it should be:

`"extends": "airbnb-base"`